scripts/ristiinvertailu/Makefile

```diff
@@ -0,0 +1,83 @@
+SHELL := /bin/bash
+DATE_NOW := $(shell date '+%Y-%m-%d')
+ALKUPVM := 2024-11-13 # TODO: Change this!
+OUTPUTS_DIR := outputs
+
+ifndef AWS_PROFILE
+$(error AWS_PROFILE is not set)
+endif
+
+ifndef DDB_JAKSOTUNNUS_TABLE_NAME
+$(error DDB_JAKSOTUNNUS_TABLE_NAME is not set)
```

Comment:
Tän muuten pystyy myös automatisoimaan:

```
(myenv) atehwa@kladakong:~/proj/cloud-base$ aws dynamodb --profile oph-prod list-tables | jq -r '.TableNames[]' | grep jaksotunnusTable | tail -1
sade-services-heratepalvelu-tep-jaksotunnusTable87D50EF4-9OFXIX8UIBKY
(myenv) atehwa@kladakong:~/proj/cloud-base$ aws dynamodb --profile oph-prod list-tables | jq -r '.TableNames[]' | grep AMISHerateTable | tail -1
sade-services-heratepalvelu-AMISHerateTable717F426F-H8X6XLMKDFIP
```

Mut kieltämättä tää on vähän epävarmaa, koska meillä on niitä ihme jäännetauluja sekä tuotannossa että testissä.  Ne pitäis oikeesti poistaa.